scw/errors.go

```diff
@@ -339,3 +340,18 @@ func (e ConfigFileNotFoundError) IsScwSdkError() {}
 func (e ConfigFileNotFoundError) Error() string {
 	return fmt.Sprintf("scaleway-sdk-go: cannot read config file %s: no such file or directory", e.path)
 }
+
+// ResourceExpiredError implements the SdkError interface
+type ResourceExpiredError struct {
```

Comment:
the error must be added in `unmarshalStandardError`

Reply:
done